api-service/job-server-demo/package.json

```diff
@@ -5,7 +5,7 @@
   "version": "0.0.1",
   "license": "Apache-2.0",
   "scripts": {
-    "setup": "mkdir -p build/server/ && mkdir -p build/config/ && rsync --ignore-existing server_config.template.json build/config/server_config.json",
+    "setup": "mkdir -p build/server/ && mkdir -p build/static/ && mkdir -p build/config/ && rsync --ignore-existing server_config.template.json build/config/server_config.json",
```

Comment:
You could add mkdir build/static here so the user doesn't need to do it manually.

Reply:
done.